src/tools/info/sys_info.c

```diff
@@ -72,7 +74,7 @@ void print_sys_info()
 
     printf("# Timer frequency: %.3f MHz\n", ucs_get_cpu_clocks_per_sec() / 1e6);
     printf("# CPU model: %s\n", cpu_model_names[ucs_arch_get_cpu_model()]);
-
+    ucs_global_opts_arch_print_memcpy_limits(&ucs_global_opts.arch);
```

Comment:
rename to:
ucs_arch_print_memcpy_limits

Reply:
renamed